### ISSUE TITLE
Set black text on appropriate homescreen buttons

### ISSUE THREAD

**Issue by hawkrives** [StoDevX/AAO-React-Native#2253]
One | Two
--- | ---
![GitHawk Upload by hawkrives](https://i.imgur.com/AeAhQRh.jpg) | ![GitHawk Upload by hawkrives](https://i.imgur.com/OiUmDC3.jpg)

iOS is pushing us to use black text on the SIS, Orgs, and Moodle buttons.
